src/main/resources/assessment-config/test/test-assessment-v1.yml

```diff
@@ -0,0 +1,220 @@
+version: 99
```

Comment:
Looks good but could you move this to `src/test/resources/assessment-config/test-assessment-v1.yml` just so there's no chance of this ever being used by the real API?

Reply:
moved as suggested